patterns/services-team-photos.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Title: Services - Team photos
+ * Slug: twentytwentyfive/services-team-photos
+ * Categories: banner, call-to-action, featured
+ * Description: Team photos in services section.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"align":"wide","layout":{"type":"default"}} -->
+<div class="wp-block-group alignwide"><!-- wp:columns -->
+<div class="wp-block-columns"><!-- wp:column -->
+<div class="wp-block-column"><!-- wp:heading {"level":3} -->
+<h3 class="wp-block-heading">Our small team is a group of driven, detail-oriented people who are passionate about their customers.</h3>
+<!-- /wp:heading --></div>
+<!-- /wp:column -->
+
+<!-- wp:column -->
+<div class="wp-block-column"><!-- wp:columns -->
+<div class="wp-block-columns"><!-- wp:column -->
+<div class="wp-block-column"><!-- wp:image {"aspectRatio":"1","scale":"cover","sizeSlug":"full","linkDestination":"none"} -->
+<figure class="wp-block-image size-full"><img src="" alt="" class="wp-image-11" style="aspect-ratio:1;object-fit:cover"/></figure>
```

Comment:
@devmuhib009, could you please remove the wp-image-11 class from all four placeholders?